tests/js/client/load-balancing/load-balancing-key-generator-noauth-cluster.js

```diff
@@ -90,18 +90,60 @@ function KeyGeneratorSuite () {
     return res;
   }
 
+  function generateCollectionDocs(name) {
+    // check that the generated keys are sequential when we send the requests
+    // via multiple coordinators.
+    db._create(name, {keyOptions: {type: "autoincrement"}});
+    assertNotEqual("", db[name].properties().distributeShardsLike);
+    let lastKey = null;
+    let url = "/_db/" + cn + "/_api/document/" + name;
+    // send documents to both coordinators
+    let i = 0;
+    let startingKey = null;
+    if (Number(name[name.length - 1]) === 1) {
+      startingKey = "abc123";
+    } else if (Number(name[name.length - 1]) === 2) {
+      startingKey = "150000";
+    }
+    for (i; i < 100000; ++i) {
+      let doc = {};
+      if (startingKey === null || i > 0) {
+        doc = {"x": 0, "c": 1};
+      } else {
+        doc = {"_key": `${startingKey}`, "x": 0, "c": 1};
+      }
+      let result = sendRequest('POST', url, /*payload*/ {"x": 0, "c": 1}, {}, i % 2 === 0);
+      assertEqual(result.status, 202);
+      let key = result.body._key;
+      assertTrue(Number(key) === Number(lastKey) + 1 || lastKey === null, {key, lastKey});
+      lastKey = key;
+    }
+    let res = db._query(`FOR doc IN ${name} FILTER doc.x == 0 RETURN doc`).toArray();
+    assertTrue(res.length === 100000);
+    lastKey = null;
+    for (i = 0; i < res.length; ++i) {
+      let key = res[i].key;
+      assertTrue(Number(key) === Number(lastKey) + 1 || lastKey === null, {key, lastKey});
+      assertTrue(res[i].hasOwnProperty("x"));
+      assertTrue(res[i].x === 0);
+      assertTrue(res[i].hasOwnProperty("c"));
+      assertTrue(res[i].c === 1);
+      lastKey = key;
+    }
```

Comment:
First of all, the query is unsafe, because it does not guarantee the order of the results without a SORT.
And then I think we don't need this query for anything, as we are already validating the generated keys directly on insert.